### ISSUE TITLE
Provide an example of composing index-page with assets bundle

### ISSUE THREAD

**Issue by victornoel** [palantir/dropwizard-index-page#38]
Hi,

I'm not sure how to express that as an issue, but I guess the objective of `IndexPageBundle` is to be used in conjunction with something like `ConfiguredAssetsBundle` to serve the static elements needed by the index page.
Is there any good example of that? I just can't make it works :)

It would be great if the example in the repository included such use case!

**Comment by victornoel** [comment]
To be clear, my use case is that I wand IndexPage to answer all accesses to ANY resources that are not already served by `ConfiguredAssetsBundle`…

**Comment by qinfchen** [comment]
You are correct in that this bundle needs to be used in conjunction with `ConfiguredAssetsBundle`. As long as you provide all url mappings when the index page bundle is added in your app. you should be all set.  what is the issue you are running into?

**Comment by victornoel** [comment]
It took me some time to make it work (but not completely as I would have like, see below) after I created the issue, I think it would be great to have some examples thus :)

For my specific case, my problem is the following.
I have this hierarchy of files for the frontend of my application (accessible from the classpath):

```
frontend
├── assets
│   ├── i18n
│   │   ├── en.json
│   │   └── fr.json
│   └── img
│       ├── logo_petalscockpit.png
│       └── wallpaper-header.png
├── index.html
├── inline.js
├── main.7941bdd7feae71f020a8.bundle.js
├── main.7941bdd7feae71f020a8.bundle.js.gz
├── scripts.ed267d13d939140f29da.bundle.js
├── scripts.ed267d13d939140f29da.bundle.js.gz
├── styles.42a8942a380dfbf24f0c.bundle.js
└── styles.42a8942a380dfbf24f0c.bundle.js.gz
```

So I configured dropwizard like this (`/cockpit*` and `/login` are two URLs understood by the frontend application, i.e., what is loaded by the `index.html`)

``` java
bootstrap.addBundle(new ConfiguredAssetsBundle(ImmutableMap.of("/frontend/", "/")));
// TODO this is not the best because every new prefix must be added... if not, the static asset 
// servlet will take over instead of returning index.html
bootstrap.addBundle(new IndexPageBundle("frontend/index.html", ImmutableSet.of("/login", "/cockpit", "/cockpit/*")));
```

My problem is that I am not happy with the fact that I have to specify one and every prefix my application is meant to be handling: I would like to simply say that every path not matched by the static assets are valid for the index page. But I'm not sure it is possible to handle that in `IndexPageBundle` without serving the static assets too with it (because of the way servlets work).
